src/ledger-iframe-bridge.ts

```diff
@@ -122,6 +137,30 @@ export class LedgerIframeBridge implements LedgerBridge {
     }
   }
 
+  async deserializeData(
+    serializeData: LedgerBridgeSerializeData,
+  ): Promise<void> {
+    this.bridgeUrl = (serializeData.bridgeUrl as string) ?? this.bridgeUrl;
+  }
+
+  async serializeData(): Promise<LedgerBridgeSerializeData> {
+    return {
+      bridgeUrl: this.bridgeUrl,
+    };
+  }
```

Comment:
I'm a bit confused, are these used to get / set options? what is their purpose compared to what `getOptions` and `setOptions` do?

Reply:
Sorry for confusing

i thought serializeData and deserializeData is just a data getter setter? 

which they may not the same data as your bridge configuration?

e.g you init a bridge with some custom configuration, but you dont need this configuration return in serializeData (e.g database connection string), as serializeData is a hash/struct for the keyring itself, bridge is a communication channel / api interface to perform operation

setOptions is something that will kick on side effect action

e.g i set a bridge URL which will reload the iframe